test/tests/workspace/mount_proc_test.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package workspace
+
+import (
+	"context"
+	"fmt"
+	"net/rpc"
+	"sync"
+	"testing"
+	"time"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+
+	agent "github.com/gitpod-io/gitpod/test/pkg/agent/workspace/api"
+	"github.com/gitpod-io/gitpod/test/pkg/integration"
+)
+
+const (
+	numberOfMount = 500
+	parallel      = 5
```

Comment:
There was no material to determine if it was a reasonable number. Do you have a good idea?
If it takes too long and becomes a problem, I'd consider adding an annotation for stress testing.

Reply:
I was going to go with 1000 at first, but I'm having trouble finding any evidence that 1000 and 500 are different. The larger the number, the longer it takes to test.

Reply:
Then lets go with 500 for now. If we see this problem again and the test does not show it, we can increase the number of iterations.

Reply:
👍

Reply:
@Furisto Thanks for your review!